### ISSUE TITLE
Bullet lists with RTL 

### ISSUE THREAD

**Issue by KhurramShehzadd** [tinymce/tinymce#5355]
When i try to apply bullets to content which is based in RTL. The bullets don't shown properly as it is shown when i'm using LTR. I think there is some problem with bullet margins when using RTL. Can you please check and update ? Attached below is screenshot. 

Currently I'm working on 5.1.5. 

Any help in this regard would be highly appreciated.

![tinymce issue](https://user-images.githubusercontent.com/23308945/72706758-30427f00-3b78-11ea-9dfa-ab1f632ff163.PNG)

**Comment by lostkeys** [comment]
Yes it indeed looks like we have an issue with the default RTL styles. I did some testing and we apply the directionality attributes sometimes on the <li> and sometimes on the <ul>. We'll look into this.

**Comment by lostkeys** [comment]
In the meantime you can try to add `content_style: 'li[dir=rtl] {margin-right: 2em;}'` to your TinyMCE config. Here is a [live example](http://fiddle.tinymce.com/FZgaab/1).

**Comment by KhurramShehzadd** [comment]
@lostkeys Thanks for your quick help. Would be waiting for the fix!
